src/eterna/mode/DesignBrowser/DesignBrowserMode.ts

```diff
@@ -1004,27 +1013,41 @@ export default class DesignBrowserMode extends GameMode {
                 if (category === DesignCategory.SEQUENCE) {
                     dataArray.push(singleLineRawData.sequence.sequenceString());
                     if (ii === 0) {
-                        dataCol.setWidth(singleLineRawData.sequence.length * 14
-                            + UITheme.designBrowser.dataPadding * 5);
+                        dataCol.setWidth(
+                            singleLineRawData.sequence.length * 14
+                            + UITheme.designBrowser.dataPadding * 5
+                        );
                         dataCol.drawGridText();
                     }
+                } else if (category === DesignCategory.LIBRARY_NT) {
+                    dataArray.push(singleLineRawData.libraryNT.join(','));
+                    if (ii === 0) {
+                        dataCol.setWidth(
+                            Math.max(
+                                170,
+                                // two digits and comma?
+                                Math.max(...solutions.map((sol) => sol.libraryNT.length)) * 42
```

Comment:
In retrospect, is an average of 4 digits plus comma too much? I can imagine some of the first library designs will basically be 8 PTC nucleotides

Reply:
so maybe we want that instead of 2 digits plus comma